_rules/image-not-in-acc-tree-is-decorative-e88epe.md

```diff
@@ -0,0 +1,271 @@
+
+- A web page with informative images without an [accessible name][] may conform to WCAG 2.1 Level A when the information provided by that image is available elsewhere, either on the web page itself or on a [conforming alternative version][] of the page. For example if an equivalent text is adjacent to the image, or if the text alternative is included in the [accessible name][] of a parent element.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [H67: Using null alt text and no title attribute on img elements for images that AT should ignore](https://www.w3.org/WAI/WCAG21/Techniques/html/H67.html)
+- [Understanding Success Criterion 1.1.1: Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This decorative `img` element is ignored by assistive technologies because of the `alt` attribute is empty (`""`).
+
+```html
+Happy new year!<br />
+<img src="/test-assets/shared/fireworks.jpg" alt="" />
+```
+
+#### Passed Example 2
+
+This decorative `img` element is ignored by assistive technologies because `aria-hidden` is set to `true`.
+
+```html
+Happy new year!<br />
+<img src="/test-assets/shared/fireworks.jpg" aria-hidden="true" alt="decorative" />
+```
+
+#### Passed Example 3
+
+This decorative `img` element is ignored by assistive technologies because it has an [explicit semantic role][] of `none`.
+
+```html
+Happy new year!<br />
+<img src="/test-assets/shared/fireworks.jpg" role="none" alt="ignore me" />
+```
+
+#### Passed Example 4
+
+This decorative `svg` element is ignored by assistive technologies because it has no attribute that would give it an [accessible name][].
+
+```html
+Happy new year!<br />
+<svg height="200" xmlns="http://www.w3.org/2000/svg">
+	<polygon points="100,10 40,180 190,60 10,60 160,180" fill="yellow" />
+</svg>
+```
+
+#### Passed Example 5
+
+This decorative `canvas` element is ignored by assistive technologies because it has no attribute that would give it an [accessible name][].
+
+```html
+Happy new year!<br />
+<canvas id="newyear" width="200" height="200"></canvas>
+<script>
+	const ctx = document.querySelector('#newyear').getContext('2d')
+	ctx.fillStyle = 'yellow'
+	ctx.beginPath()
+	ctx.moveTo(100, 10)
+	ctx.lineTo(40, 180)
+	ctx.lineTo(190, 60)
+	ctx.lineTo(10, 60)
+	ctx.lineTo(160, 180)
+	ctx.fill()
+</script>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `img` element with an empty (`""`) `alt` is not [purely decorative][].
+
+```html
+<img src="/test-assets/shared/w3c-logo.png" alt="" />
+```
+
+#### Failed Example 2
+
+This `img` element with `aria-hidden` set to `true` is not [purely decorative][].
```

Comment:
Need more desciption may be something like that:
"This `img` element is visible and not included in accessibility tree because of `aria-hidden="true"` but is not purely decorative."